### ISSUE TITLE
Feature to for better user experience 

### ISSUE THREAD

**Issue by vibhi19** [glitch-soc/mastodon#894]
Pitch

Popular Hashtag -
It will help people know what other are talking about and can take part in it and also help finding new users.
News-
its  needed as people really wanted to get updated of their surrounding and world and it will help twitter user to move from twitter.
Verified or featured user - This can be shown to New user as whom to follow.The list will be moderated by admin.it will help new user to interact to people and get content on home feed.yes there is profile directory but its simply not the right solution.
Verified profile with some type of tick. 
Doddle - add a eraser in it for god sake and Few shape and meme maker would be awesome in place of doddle

**Comment by ashkitten** [comment]
- for trending hashtags, this has been [very controversially implemented and removed shortly thereafter](tootsuite#7702)
- not really sure what you mean by "news", could you elaborate?
- you can already set a list of users for new accounts to automatically follow, as for the rest of this suggestion i am highly opposed to any sort of twitter-like "checkmark verification" - we already have staff badges on profiles, i don't see anything else this could be used for besides blatant favoritism
- if you'll notice, the background of mastodoodle is white. it does not support transparency, so you can just paint it back to white if you need. this is a small feature by @MightyPork, and if you want to work on more features for it i'm sure they'd be accepted. you can find a list of bugs related to it [here](https://github.com/glitch-soc/mastodon/labels/ft-doodles).

**Comment by yipdw** [comment]
I don't think these are improvements for glitch-soc's current userbase.  I have not done a controlled poll, but my impression is that glitch-soc's users do _not_ want trending hashtags, news items, or centrally-controlled verification (note: the `rel="me"` verification service is already part of glitch-soc).

(I'd be happy to revisit this impression, though.)

**Comment by vibhi19** [comment]
> * for trending hashtags, this has been [very controversially implemented and removed shortly thereafter](tootsuite#7702)
Can you check the link again i could not find the issue and it would be awesome if we have option to enable/disable it based on our need 
> * not really sure what you mean by "news", could you elaborate?
I meant by something like fetching rss from different site and showing it to user based on their respective category  
> * you can already set a list of users for new accounts to automatically follow, as for the rest of this suggestion i am highly opposed to any sort of twitter-like "checkmark verification" - we already have staff badges on profiles, i don't see anything else this could be used for besides blatant favoritism
Yeah badges saying verified would be good and the issue with user following automatically is just not right show them bunch of popular profile which they can follow giving more control to user
> * if you'll notice, the background of mastodoodle is white. it does not support transparency, so you can just paint it back to white if you need. this is a small feature by @MightyPork, and if you want to work on more features for it i'm sure they'd be accepted. you can find a list of bugs related to it [here](https://github.com/glitch-soc/mastodon/labels/ft-doodles).
I tried to make a doodle and it was hard so thats why i right i dont think many people uses it.

**Comment by vibhi19** [comment]
> I don't think these are improvements for glitch-soc's current userbase. I have not done a controlled poll, but my impression is that glitch-soc's users do _not_ want trending hashtags, news items, or centrally-controlled verification (note: the `rel="me"` verification service is already part of glitch-soc).
> 
> (I'd be happy to revisit this impression, though.)

what is the whole issue with trending hashtag?
News would be awesome as for new user coming from twitter its basically a dream come true and who dont like them?
its not just centrally-controlled verification i also added showing popular profile to user which new user can follow so a new user can immediately can start interaction with them.

**Comment by ashkitten** [comment]
fixed link in my comment

**Comment by yipdw** [comment]
The hashtag issue link was incorrect; please try https://github.com/tootsuite/mastodon/issues/7702.  In particular, comment https://github.com/tootsuite/mastodon/issues/7702#issuecomment-393993386 gives some reasons.

(As a tangent: my _personal_ belief is that this mechanism encourages an environment where people feel like they are encouraged to jump into conversations and topics that are labelled as "popular", regardless of the depth or correctness of their understanding.  You can see other arguments on the linked thread.)

If you want news updates, there are many bots that already provide that, and you can follow them to get updates (e.g. there's a BBC headline repeater at https://bot.undernet.uy/bbc; plug `@bbc@bot.undernet.uy` into the search box of your local instance to retrieve the user and follow it).

I don't think that "immediately starting interaction with someone" is a good thing on its face; to make that conclusion, we'd have to adopt the assumption that the goal of a social network is to maximize interactions between all (or at least newly-added) parties.  However, I think many people who currently use glitch-soc and glitch-soc instances use the software to communicate with people they know, reaching out to new people at their own pace.

**Comment by ThibG** [comment]
First, in the future, please open one issue per requested feature, that makes it much easier to discuss.

- Trending hashtags: I do not think they are worth the downsides discussed in the linked issues. Furthermore, they would be of limited use in a decentralized setting (unless you are running on a large instance)
- News: I'm not too sure what you are suggesting here. I guess following RSS or atom feeds would be somewhat useful, but it seems slightly out of scope for Mastodon/glitch-soc. As pointed out, bots kind of fill the gap here.
- Verified users: who would verify which user? Users can already prove ownership of a remote web page, I think that is enough.
- Featured users: there are two existing mechanisms for that: individuals endorsing other users, and admins setting up a list of people to follow on account creation. The first feature is only visible on the public pages, not from within the Web UI. Maybe it could be made a little more prominent.

**Comment by vibhi19** [comment]
> First, in the future, please open one issue per requested feature, that makes it much easier to discuss.

hmm I will do I am kind of new so not aware of whole things

> * Trending hashtags: I do not think they are worth the downsides discussed in the linked issues. Furthermore, they would be of limited use in a decentralized setting (unless you are running on a large instance)

its just over exaggeration there are other Activity pub based network like osada which show them without an issue so at least giving an option would be good at least Give power to admin enable and disable plus make it only for local user. 

> * News: I'm not too sure what you are suggesting here. I guess following RSS or atom feeds would be somewhat useful, but it seems slightly out of scope for Mastodon/glitch-soc. As pointed out, bots kind of fill the gap here.

bot feel more like spam then actual news. an column for news would be awesome with different feature alot can be discussed over it  

> * Verified users: who would verify which user? Users can already prove ownership of a remote web page, I think that is enough.

a badge would not hurt any one like giving them to popular users its a win win for everyone 

> * Featured users: there are two existing mechanisms for that: individuals endorsing other users, and admins setting up a list of people to follow on account creation. The first feature is only visible on the public pages, not from within the Web UI. Maybe it could be made a little more prominent.

already said automatic follow doesn't  feel right I want something like that tutorial on how to user after that you can showcase bunch of user to follow.

**Comment by ThibG** [comment]
I do not believe trending hashtags, featured or verified users are things the glitch-soc project is interested in.
I'm not sure about the “News” part either, I'm not sure what you would like it to be exactly, and I think this use case could be covered by using a list.
